beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -414,30 +414,30 @@ impl From<execution_layer::Error> for ExecutionPayloadError {
     }
 }
 
-impl<E: EthSpec> From<ExecutionPayloadError> for BlockError<E> {
+impl From<ExecutionPayloadError> for BlockError {
     fn from(e: ExecutionPayloadError) -> Self {
         BlockError::ExecutionPayloadError(e)
     }
 }
 
-impl<E: EthSpec> From<InconsistentFork> for BlockError<E> {
+impl From<InconsistentFork> for BlockError {
     fn from(e: InconsistentFork) -> Self {
         BlockError::InconsistentFork(e)
     }
 }
 
-impl<E: EthSpec> std::fmt::Display for BlockError<E> {
+impl std::fmt::Display for BlockError {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            BlockError::ParentUnknown(block) => {
-                write!(f, "ParentUnknown(parent_root:{})", block.parent_root())
+            BlockError::ParentUnknown { parent_root, .. } => {
```

Comment:
This special case handling shouldn't be necessary anymore

Reply:
Same as above, can derive display now